UnityProject/Assets/Scripts/Items/ClothingItem.cs

```diff
@@ -8,13 +9,14 @@ public enum SpriteHandType
 }
 
 [RequireComponent(typeof(SpriteRenderer))]
-public class ClothingItem : MonoBehaviour
+public class ClothingItem : NetworkBehaviour
 {
 	/// <summary>
 	/// Absolute orientation
 	/// </summary>
 	private Orientation currentDirection = Orientation.Down;
-	public int reference = -1;
+	[SyncVar(hook = nameof(SyncSprite))] public int reference = -1;
+	[SyncVar(hook = nameof(SyncColor))] public Color color;
```

Comment:
try
```suggestion
	[SyncVar(hook = nameof(SyncColor))] private string hexColor;
	public Color color;
```
and then use `DebugTools.ColorToHex( color )` and `DebugTools.HexToColor( hexColor )`